commands/call.js

```diff
@@ -1,5 +1,5 @@
-const nearlib = require('near-api-js');
-const { utils } = nearlib;
+const nearAPI = require('near-api-js');
+const { utils } = nearAPI;
```

Comment:
should we just change this to:

```js
const { utils, providers } = require('near-api-js');
```

I think it's preferable style to `nearAPI`